### DISCUSSION TITLE
Support to save History for Bing Chat at Microsoft Edge Dev Sidebar

### DISCUSSION THREAD

**Discussion by hoanghxn** [benf2004/Bing-Chat-History#5] [Ideas]
I often use Binh chat at microsoft edge Dev with sidebar feature which is very convenient during research. Hope Bing Chat History will support saving History from here 🙏

**Comment by hoanghxn** [comment]
It would also be nice to have the option of Bing Chat History to be saved to Notion

**Comment by benf2004** [comment]
Hi @hoanghxn. Thank you for the suggestions!

I have done some testing on trying to access the sidebar in Edge and it has proven to be difficult. It's actually just an iframe of this website: https://edgeservices.bing.com/edgediscover/query?. However, putting a content script into this website has not worked. I will keep trying and let you know if I figure it out. 

As for Notion, I do not use Notion (I use Obsidian MD), but I suspect it will not be too difficult. I will try to find some docs and see what I can do this week. In the meantime, you can check out the export as markdown option.

**Reply by hoanghxn** [reply]
Thanks you, With ChatGPT I'm using "**[ChatGPT to Notion](https://github.com/L-a-r-t/chatgpt-to-notion)**" to save to Notion for group members to refer to.
